pkg/whitesource/configHelper_test.go

```diff
@@ -0,0 +1,256 @@
+}
+
+func TestAddGeneralDefaults(t *testing.T) {
+	t.Parallel()
+
+	utilsMock := NewScanUtilsMock()
+
+	t.Run("default", func(t *testing.T) {
+		testConfig := ConfigOptions{}
+		whitesourceConfig := ScanOptions{
+			OrgToken:       "testOrgToken",
+			ProductName:    "Test",
+			ProductToken:   "testProductToken",
+			ProductVersion: "testVersion",
+			ProjectName:    "testProject",
+			UserToken:      "testuserKey",
+		}
+		testConfig.addGeneralDefaults(&whitesourceConfig, utilsMock)
+		assert.Equal(t, "checkPolicies", testConfig[0].Name)
+		assert.Equal(t, true, testConfig[0].Value)
+		assert.Equal(t, "forceCheckAllDependencies", testConfig[1].Name)
+		assert.Equal(t, true, testConfig[1].Value)
+
+		assert.Equal(t, "testOrgToken", testConfig[2].Value)
+		assert.Equal(t, "Test", testConfig[3].Value)
+		assert.Equal(t, "testVersion", testConfig[4].Value)
+		assert.Equal(t, "testProject", testConfig[5].Value)
+		assert.Equal(t, "testVersion", testConfig[6].Value)
+		assert.Equal(t, "testProductToken", testConfig[7].Value)
+		assert.Equal(t, "testuserKey", testConfig[8].Value)
+	})
+
+	t.Run("DIST product", func(t *testing.T) {
+		testConfig := ConfigOptions{}
+		whitesourceConfig := ScanOptions{
+			OrgToken:       "testOrgToken",
+			ProductName:    "DIST - Test",
+			ProductToken:   "testProductToken",
+			ProductVersion: "testVersion",
+			ProjectName:    "testProject",
+			UserToken:      "testuserKey",
+		}
+		testConfig.addGeneralDefaults(&whitesourceConfig, utilsMock)
+		assert.Equal(t, "checkPolicies", testConfig[0].Name)
+		assert.Equal(t, false, testConfig[0].Value)
+		assert.Equal(t, "forceCheckAllDependencies", testConfig[1].Name)
+		assert.Equal(t, false, testConfig[1].Value)
+	})
+
+	t.Run("verbose", func(t *testing.T) {
+		testConfig := ConfigOptions{}
+		whitesourceConfig := ScanOptions{
+			Verbose: true,
+		}
+		testConfig.addGeneralDefaults(&whitesourceConfig, utilsMock)
+		assert.Equal(t, "log.level", testConfig[2].Name)
+		assert.Equal(t, "debug", testConfig[2].Value)
+		assert.Equal(t, "log.files.level", testConfig[3].Name)
+		assert.Equal(t, "debug", testConfig[3].Value)
+	})
+
+	t.Run("includes and excludes", func(t *testing.T) {
+		testConfig := ConfigOptions{}
+		whitesourceConfig := ScanOptions{
+			Excludes: []string{"**/excludes1", "**/excludes2"},
+			Includes: []string{"**/includes1", "**/includes2"},
+		}
+		testConfig.addGeneralDefaults(&whitesourceConfig, utilsMock)
+		assert.Equal(t, "excludes", testConfig[2].Name)
+		assert.Equal(t, "**/excludes1 **/excludes2", testConfig[2].Value)
+		assert.Equal(t, true, testConfig[2].Force)
+		assert.Equal(t, "includes", testConfig[3].Name)
+		assert.Equal(t, "**/includes1 **/includes2", testConfig[3].Value)
+		assert.Equal(t, true, testConfig[3].Force)
+	})
+}
+
+func TestAddBuildToolDefaults(t *testing.T) {
+	t.Parallel()
+
+	t.Run("success case", func(t *testing.T) {
+		utilsMock := NewScanUtilsMock()
```

Comment:
Just FYI, you have to also invoke t.Parallel() within every Run(), in addition to line 167, for them to run in parallel.
```suggestion
		t.Parallel()
		utilsMock := NewScanUtilsMock()
```